components/tools/OmeroPy/test/integration/test_tickets3000.py

```diff
@@ -68,11 +68,6 @@ def test2396(self):
         l_ia.setChild(fa)
         self.update.saveObject(l_ia)
 
-    # This test is no longer valid as it should not be possible to remove
```

Comment:
Implemented in 5764c79f79f4ca84cf3e265781936b50c4ef0f5f in relation to http://trac.openmicroscopy.org.uk/ome/ticket/2547. The logic of the test seems valid since the user is added to a new group before the call to `removeGroups()`. Either un-marking this test as broken or simply removing it could be the way forward. /cc @joshmoore